src/Education/Resources.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="Education_NoRuleInfo" xml:space="preserve">
+    <value>[Education] Unable to locate help for rule '{0}'. Help should be available at https://rules.sonarsource.com.</value>
```

Comment:
This message is pretty much as the one that used to be returned by `IRuleMetadataProvider`.

The difference is that this message is only shown in the Output Window, whereas previously the tool window document was updated.

However, updating the tool window would take more work; this case shouldn't really occur anyway, we're just being cautious; and we're trying to finish of the sprint, so we should be minimising the amount of change at this stage.